app/src/app/job.rs

```diff
@@ -47,13 +40,13 @@ pub trait JobCacheKeys {
     }
 }
 #[async_trait]
-pub trait JobApp: Send + Sync {
+pub trait JobApp: fmt::Debug + Send + Sync {
     #[allow(clippy::too_many_arguments)]
     async fn enqueue_job(
         &self,
         worker_id: Option<&WorkerId>,
         worker_name: Option<&String>,
-        arg: Option<RunnerArg>,
+        arg: Vec<u8>,
```

Comment:
all arg is defined in runner_schema.job_arg_proto (decoded by each runner internally)